src/main/java/be/uantwerpen/scicraft/ScicraftClient.java

```diff
@@ -57,20 +73,43 @@ public void onInitializeClient() {
         // Register rendering for electron entity
         EntityRendererRegistry.register(Entities.ELECTRON_ENTITY, FlyingItemEntityRenderer::new);
         EntityRendererRegistry.register(Entities.PROTON_ENTITY, FlyingItemEntityRenderer::new);
+        EntityRendererRegistry.register(Entities.ANTI_PROTON_ENTITY, FlyingItemEntityRenderer::new);
         EntityRendererRegistry.register(Entities.NEUTRON_ENTITY, FlyingItemEntityRenderer::new);
+        EntityRendererRegistry.register(Entities.ANTI_NEUTRON_ENTITY, FlyingItemEntityRenderer::new);
         EntityRendererRegistry.register(Entities.ENTROPY_CREEPER, EntropyCreeperEntityRenderer::new);
 
+        BlockEntityRendererRegistry.register(BlockEntities.BOHR_BLOCK_ENTITY, BohrBlockEntityRenderer::new);
+
         BlockEntityRendererRegistry.register(BlockEntities.ANIMATED_CHARGED_BLOCK_ENTITY, ChargedBlockEntityRenderer::new);
         BlockEntityRendererRegistry.register(BlockEntities.CHARGED_PLACEHOLDER_BLOCK_ENTITY, ChargedPlaceholderBlockEntityRenderer::new);
         BlockEntityRendererRegistry.register(BlockEntities.ELECTRIC_FIELD_SENSOR, ElectricFieldSensorRenderer::new);
 
         // Register rendering lewis crafting table inventory
         HandledScreens.register(ScreenHandlers.LEWIS_SCREEN_HANDLER, LewisScreen::new);
+
         ScreenEvents.BEFORE_INIT.register((a, screen, b, c) -> {
             if (screen instanceof LewisScreen)
                 ScreenMouseEvents.afterMouseRelease(screen).register((d, mouseX, mouseY, e) -> ((LewisScreen) screen).getButtonWidget().onClick(mouseX, mouseY));
         });
 
+//        ScreenEvents.BEFORE_INIT.register((a, screen, b, c) -> {
+//            if (screen instanceof LewisScreen)
+//                ScreenMouseEvents.afterMouseRelease(screen).register((d, mouseX, mouseY, e) -> ((LewisScreen) screen).getButtonWidget().onClick(mouseX, mouseY));
+//        ClientPlayNetworking.registerGlobalReceiver(NetworkingConstants.CHARGED_MOVE_STATE, (client, handler, buf, responseSender) -> {
+//            BlockPos target = buf.readBlockPos();
+//            String block_name = buf.readString();
+//            boolean annihilation = buf.readBoolean();
+//            client.execute(() -> {
+//                if (client.world != null) {
+//                    if (client.world.getBlockEntity(target) instanceof AnimatedChargedBlockEntity particle2) {
+//                        particle2.render_state = particle2.string2BlockState(block_name);
+//                        particle2.annihilation = annihilation;
+//                    }
+//                }
+//            });
+//        }
+//        );
+
```

Comment:
Why is this commented out? Might give problems with other features if this gets into main branch.

Reply:
If i remember correctly, this code was not present in main anymore and we got a conflict so we commented this and forgot to remove it later